Luke_Smith_Portfolio.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width initial-scale=1">
+	<title>Luke Smith Portfolio Home</title>
+    <link rel="stylesheet" href="Luke_Smith_Portfolio.css">
+</head>
+<body>
+	<div class="topnav">
+	<div class="center">
+		<ul>
+		<li><a href="#home">HOME</a></li>
+  	  	<li><a href="Luke_Smith_Portfolio_About_Me.html">ABOUT ME</a></li>
+  	  	<li><a href="Luke_Smith_Portfolio_Portfolio.html">PORTFOLIO</a></li>
+  	  	<li><a href="Luke_Smith_Portfolio_Contact.html">CONTACT</a></li>
+        </ul>
+	<div>
+  	    <img src="Luke_Smith_Logo.png" alt="Luke Logo" style="width:100%">
```

Comment:
The style="width:100%" here is bad practice. Styling for this image should be done in the stylesheet using e.g.
img {
        width: 100%;
}

I think you actually have a styling rule to this effect, so just delete the style attribute on this tag